File: Coins2/app/src/main/java/com/lq/coins/ApiService.java
package com.lq.coins;


import java.util.List;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface ApiService {

    @GET("atilsamancioglu/K21-JSONDataSet/master/crypto.json")
    Observable<List<CoinModel>> getAll();

}

File: CryptoAppWithCompose/app/src/main/java/com/lq/cryptoappwithcompose/service/ApiService.kt
package com.lq.cryptoappwithcompose.service

import com.lq.cryptoappwithcompose.modal.CryptoModel
import retrofit2.Call
import retrofit2.http.GET

interface ApiService {
    @GET(value = "atilsamancioglu/K21-JSONDataSet/master/crypto.json")
    fun getAllCrypto() : Call<List<CryptoModel>>;

}